contact_marc.html

```diff
@@ -45,15 +45,15 @@
                 <h1 class="heroText">Blue Orchid</h1>
                 <p class="heroTag">Rare Plant Society</p>
                 <!--Button! about us-->
-            <button type="button" class="btn btn-circle btn-lg">Home</button>
+            <button type="button" class="btn btn-circle btn-lg" alt="In the backgroud we have for the entire page a picture of our blue, lightblue Orchid">Home</button>
```

Comment:
I don't think the description of the background belongs in the button element.  Maybe find a better place for it?